utilities/src/main/java/io/syndesis/qe/resource/impl/Ops.java

```diff
@@ -0,0 +1,204 @@
+            .withScope(crd.getSpec().getScope())
+            .withVersion(crd.getSpec().getVersion());
+        return builder.build();
+    }
+
+    private NamespacedOpenShiftClient getMonitoringNamespace() {
+        return OpenShiftUtils.getInstance().inNamespace(NAMESPACE);
+    }
+
+    /*
+     *  application monitoring gets stuck when deleting a namespace when a finalizer is present in the metadata
+     */
+    private void finalizerWorkaround() {
+        RawCustomResourceOperationsImpl crClients = OpenShiftUtils.getInstance().customResource(getApplicationMonitoringContext());
+        List<Object> items = (List<Object>) crClients.list().get("items");
+        items.forEach(item -> {
+            try {
+                Map<String, Object> cr = (Map<String, Object>) item;
+                Map<String, Object> metadata = (Map<String, Object>) cr.get("metadata");
+                String name = (String) metadata.get("name");
+                String namespace = (String) metadata.get("namespace");
+                metadata.remove("finalizers");
+                crClients.edit(namespace, name, cr);
+            } catch (Exception e) {
+                log.error("Could not edit application monitoring CRs", e);
+            }
+        });
+    }
+
+    @Override
+    public void deploy() {
+        final NamespacedOpenShiftClient ocp = getMonitoringNamespace();
+        log.info("Installing monitoring applications");
+        if (!isDeployed()) {
+            try {
+                Process p;
+                if (!new File(TMP_FOLDER).exists()) {
+                    p = new ProcessBuilder()
+                        .directory(new File("/tmp"))
+                        .redirectOutput(new File("/tmp/gitout"))
+                        .command("git", "clone", "--depth", "1", "--branch", TestConfiguration.appMonitoringVersion(),
+                                 "https://github.com/integr8ly/application-monitoring-operator")
+                        .start();
+                    p.waitFor();
+                    //Use oc used by the testsuite
+                    Path scriptPath = Paths.get(TMP_FOLDER, "scripts", "install.sh");
+                    String script = IOUtils.toString(Files.newReader(scriptPath.toFile(), Charset.defaultCharset()));
+                    String ocReplaced = script.replaceAll("oc", OpenShifts.getBinaryPath());
+                    Files.write(ocReplaced.getBytes(Charset.defaultCharset()), scriptPath.toFile());
+                }
+                //Oc can be logged in to only one cluster at a time, this ensures the OC is really logged in
+                new ProcessBuilder().command(
+                    OpenShifts.getBinaryPath(),
+                    "login",
+                    TestConfiguration.openShiftUrl(),
+                    "-u", TestConfiguration.adminUsername(),
+                    "-p", TestConfiguration.adminPassword()
+                ).start().waitFor();
```

Comment:
`OpenshiftUtils.binary().login(url, user, pass)`

Reply:
No can do, I need to do it via the process to bind the oc binary for the installation script.